File: Venkata Ramana/week 1 day2/findshoes.py
# cook your dish here
num_tests = int(input())
for t in range(num_tests) :
    D, C = map(int, input().split())
    first_day_order = sum(list(map(int, input().split())))
    second_day_order = sum(list(map(int, input().split()))) 
    
    cost_without_coupon = 2 * D 
    cost_with_coupon = C 
    if first_day_order < 150 : 
        cost_with_coupon += D 
    if second_day_order < 150 : 
        cost_with_coupon += D 
        
    if cost_with_coupon < cost_without_coupon : 
        print("YES")
    else :
        print("NO")

File: Venkata Ramana/week 1 day2/findshoes.c
#include <stdio.h>

int main(void) {
	// your code goes here
	int T;
	scanf("%d",&T);
	while(T--)
	{
	    int N,M;
	    scanf("%d %d",&N,&M);
	    if(N>M)
	    {
	        printf("%d\n",N*2-M);
	    }
	    else
	    {
	        printf("%d\n",N);
	    }
	}
	return 0;
}